MAINTAINERS

```diff
@@ -0,0 +1,17 @@
+# Org-Level Maintainers (in alphabetical order)
+# Pattern: [First Name] [Last Name] <[Email Address]> ([GitHub Handle])
+Feynman Zhou <feynmanzhou@microsoft.com> (@FeynmanZhou)
+Justin Cormack <justin.cormack@docker.com> (@justincormack)
+Milind Gokarn <gokarnm@amazon.com> (@gokarnm)
+Niaz Khan <niazfk@amazon.com> (@niazfk)
+Pritesh Bandi <priteshbandi@gmail.com> (@priteshbandi)
+Steve Lasker <StevenLasker@hotmail.com> (@stevelasker)
+Toddy Mladenov <toddysm@gmail.com> (@toddysm)
+Vani Rao <vaninrao@amazon.com> (@vaninrao10)
+Yi Zha <yizha1@microsoft.com> (@yizha1)
```

Comment:
General questions:
1. Should org-level maintainers be in every repository?
2. Should org-level maintainers be CODEOWNERS in every repository? It is worth noting that being a code owner means that you will be notified and requested for code review.
3. Are org-level maintainers counted for repo-level voting? e.g. Publish a new release.
4. What if an org-level maintainer is inactive for a long time for a certain repository but he/she is indeed active in other repositories?
5. Is this pattern scalable for org-level maintainers?

Reply:
The initial intent was the Org maintainers are a subset of folks to facilitate overall project support. They may not be as active in every project. 
When using GitHub teams, the permissions are configured through overlay of multiple teams, across multiple repositories. However, there's no traceability for those changes. 
So, yes, it's assumed org owners are there to collaboratively help the projects when needed. 
The reason to list org owners in each repo is redundant, as org owners are captured here: https://github.com/notaryproject/.github/blob/main/MAINTAINERS
So, IMO, org-owners are not required in each repo.

Reply:
Agreed, this should be fixed in the latest commit.